media/css/press/press.scss

```diff
@@ -0,0 +1,46 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../protocol/css/includes/lib';
+@import '../../protocol/css/components/notification-bar';
+@import '../../protocol/css/components/emphasis-box';
+
+.mzp-c-article {
+    h2 {
+        @include text-display-xs;
+    }
+}
+
+.mzp-c-emphasis-box {
+    margin-top: $spacing-xl;
+}
+
+//* -------------------------------------------------------------------------- */
+// Form
+
+fieldset {
+    padding: $spacing-sm;
+    margin-top: $spacing-lg;
+
+    label {
+        margin: $spacing-sm 0;
+    }
+}
+
+.mzp-c-button {
+    margin-top: $spacing-md;
+}
+
+.super-priority-field {
+    display: none;
+}
+
+.error-msg {
```

Comment:
The speaker request form uses a different CSS class for errors, so they are not showing up in red.

Visually, the error messages also appear closer to the previous field than the one they should be associated with, which makes it a little hard to read.

![image](https://user-images.githubusercontent.com/400117/69134398-5469dd00-0aaf-11ea-919b-8ac6331bf4af.png)

Reply:
I'm not sure how to trigger this state to debug. 🤔

Reply:
@amychurchwell you can disable native form validation in browsers by adding the `novalidate` attribute to the form locally: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#attr-novalidate